Makefile.am

```diff
@@ -4,7 +4,8 @@ AM_CFLAGS = \
 	-Werror-implicit-function-declaration \
 	-Wformat -Wformat-security -Werror=format-security \
 	-Wno-conversion -Wunused-variable -Wunreachable-code \
-	-Wall -W -D_FORTIFY_SOURCE=2 -std=c11
+	-Wall -W -D_FORTIFY_SOURCE=2 -std=c11 \
+	`pkg-config --cflags --libs libsystemd`
```

Comment:
no, you need to modify `configure.ac` to add a configure check for systemd. And then use LIBSYSTEMD_LIBS/CFLAGS instead in Makefile.am. Look at some of the other projects to see template code for that.